tests/MagentoHackathon/Composer/Magento/FullStack/AbstractTest.php

```diff
@@ -38,7 +38,7 @@ public static function setUpBeforeClass()
         @unlink(self::getBasePath().'/magento-modules/vendor/theseer/directoryscanner/tests/_data/nested/empty');
         
         $process = new Process(
-            self::getComposerCommand().' archive --format=zip --dir="tests/FullStackTest/artifact" -vvv',
+            [self::getComposerCommand().' archive --format=zip --dir="tests/FullStackTest/artifact" -vvv'],
```

Comment:
With this change we need to set minimal version for symfony/process in composer json as v2.x expect type string as argument

Reply:
I changed minimal version for symfony/process in composer json for version where attribute in construct method has new type (string|array).

Reply:
@xmav Our constraint allow to install  any versions. I checked with minimum versions for me. Composer installed the symfony/process v5.3.7. I think we can leave as is.